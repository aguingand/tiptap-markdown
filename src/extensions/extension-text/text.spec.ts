import { expect, test } from "vitest";
import { parse, serialize } from "../../../__tests__/utils";

test('parse', () => {
    expect(parse('example')).toMatchSnapshot();
});
test('parse soft break', () => {
    expect(parse('example1\nexample2')).toMatchSnapshot();
});
test.skip('serialize', () => {
    expect(serialize('example')).toEqual('example');
});
test.skip('serialize escaped', () => {
    expect(serialize('example <><>')).toEqual('example &lt;&gt;&lt;&gt;');
});
