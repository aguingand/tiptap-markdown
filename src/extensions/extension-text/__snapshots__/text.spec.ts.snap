// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`parse 1`] = `
[
  {
    "content": [
      {
        "text": "example",
        "type": "text",
      },
    ],
    "type": "paragraph",
  },
]
`;

exports[`parse soft break 1`] = `
[
  {
    "content": [
      {
        "text": "example1 example2",
        "type": "text",
      },
    ],
    "type": "paragraph",
  },
]
`;
